src/algebra/field.lean

```diff
@@ -214,46 +214,73 @@ end
 
 end
 
-namespace is_ring_hom
-open is_ring_hom
+namespace ring_hom
 
 section
-variables {β : Type*} [division_ring α] [division_ring β]
-variables (f : α → β) [is_ring_hom f] {x y : α}
+
+variables {β : Type*} [division_ring α] [division_ring β] (f : α →+* β) {x y : α}
 
 lemma map_ne_zero : f x ≠ 0 ↔ x ≠ 0 :=
-⟨mt $ λ h, h.symm ▸ map_zero f,
- λ x0 h, one_ne_zero $ calc
-    1 = f (x * x⁻¹) : by rw [mul_inv_cancel x0, map_one f]
-  ... = 0 : by rw [map_mul f, h, zero_mul]⟩
+⟨mt $ λ h, h.symm ▸ f.map_zero,
```

Comment:
Should all these `map_*` lemmas be `simp`?

Reply:
I don't know. I guess they were not marked as `simp` because looking for `is_ring_hom` is expensive.

Reply:
So now, when we bundle, we can (and should?)

Reply:
Added some `@[simp]` attrs.